docs/version0.44.md

```diff
@@ -32,6 +32,7 @@ The following new features and notable changes since version 0.43.0 are included
 - [New `-XX:[+|-]ShowUnmountedThreadStacks` option added](#new-xx-showunmountedthreadstacks-option-added)
 - [VMID query in the `jcmd` tool enhanced](#vmid-query-in-the-jcmd-tool-enhanced)
 - [DDR field names in `J9BuildFlags` changed](#ddr-field-names-in-j9buildflags-changed)
+- [Deletion of the files within the `/tmp/` folder by `systemd`controlled](#deletion-of-the-files-within-the-tmp-folder-by-systemd-controlled)
```

Comment:
Not sure what "by `systemd`controlled" means. Is "controlled" a typo or it just shouldn't be there at all.